docs/docs/solidity_equivalents/addresses.mdx

```diff
@@ -1,3 +1,12 @@
 ---
-title: Addresses (WIP)
+title: Addresses
 ---
+
+Addresses on Starknet are larger than on Ethereum.
+On Starknet they take up the whole felt space (251 bits) rather than 160 bits.
+
+To support StarkNet addresses we have slightly modified the solc compiler.
+As a result, addresses in our flavour of Solidity are represented with 256 bits instead of 160 bits.
+
+It is also important to note that the `ecrecover` precompile will not work with StarkNet addresses,
```

Comment:
```suggestion
It is also important to note that the `ecrecover` precompile does not work with StarkNet addresses since it uses Ethereum's curve. The return type of `ercrecover` has been changed from `address` to `uint160` to reflect this. 
```

Reply:
That said, should we change it StarkNet's curve since we've changed address anyway?